res/database/Default/database_news.xml

```diff
@@ -1975,10 +2297,12 @@ Demonstrations are expected again this year.</en>
 			<title>
 				<de>Wildschwein-Urteil gefällt</de>
 				<en>Wild boar judgement passed</en>
+				<pl>Wyrok w sprawie dzika wydany</pl>
 			</title>
 			<description>
 				<de>Die Bachenhausener Strafkammer befand den Gärtner Rico Mielke der Körperverletzung schuldig. „Sie sind wohl blind?“, fragte der klagende Anwalt.</de>
 				<en>The Bachenhausen Criminal Court found gardener Rico Mielke guilty of assault. "Are you blind?" asked the plaintiff lawyer.</en>
+				<pl>Sąd karny w Bachenhausen uznał ogrodnika Rico Mielke winnym napaści. "Jesteś ślepy?" zapytał adwokat powoda.</pl>
```

Comment:
Which quotes do you normally use? "these" or „these“.
I saw in some other database files that they were not used consistently.
The best would be to try for consistent usage from the very beginning (in database and property files).

Reply:
I translate from English, so as it is in English translation so I translate. If “this” then “this”. I didn't expect it to be important.